### DISCUSSION TITLE
Internal Server Error in "git push"

### DISCUSSION THREAD

**Discussion by smontanaro** [community/community#22455] [New to GitHub]
<p>I’m trying to push a couple small edits from my dev branch to GitHub, but keep getting an internal server error:</p>
<blockquote>
<p>% git push<br>
Enumerating objects: 13, done.<br>
Counting objects: 100% (13/13), done.<br>
Delta compression using up to 8 threads<br>
Compressing objects: 100% (7/7), done.<br>
Writing objects: 100% (7/7), 588 bytes | 294.00 KiB/s, done.<br>
Total 7 (delta 6), reused 0 (delta 0)<br>
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.<br>
remote: Internal Server Error<br>
To <a href="http://github.com" rel="noopener nofollow ugc">github.com</a>:smontanaro/smontanaro-net.git<br>
! [remote rejected] main → main (Internal Server Error)<br>
error: failed to push some refs to ‘git@github.com:smontanaro/smontanaro-net.git’</p>
</blockquote>
<p>I’ve done this successfully hundreds of times with no problems and see nothing to indicate the source of the error. Is there somewhere to find details in my GitHub repo?</p>

**Comment by trentwiles** [comment]
<p>Same, getting this error too:</p>
<pre><code class="lang-auto">Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 8 threads
Compressing objects: 100% (6/6), done.
Writing objects: 100% (7/7), 1.29 KiB | 1.29 MiB/s, done.
Total 7 (delta 0), reused 0 (delta 0), pack-reused 0
remote: Internal Server Error
To github.com:RiversideRocks/SearxForward.git
 ! [remote rejected] main -&gt; main (Internal Server Error)
error: failed to push some refs to 'github.com:RiversideRocks/SearxForward.git'
</code></pre>

**Comment by byrneh** [comment, marked as answer]
<p>It would be worth trying again when <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a> says the current incident is resolved. In case these errors are related to the incident.</p>

**Reply by mustafaabobakr** [reply]
thank you, i thought it was in my end
<img width="1092" height="1115" alt="image" src="https://github.com/user-attachments/assets/e8284a1e-2454-47ac-9550-ebbc5d1cd16e" />

**Comment by CesarDavidMorenR** [comment]
<p>I have the same error. Probably a bug of GitHub</p>

**Comment by smontanaro** [comment]
<p>Oh, cool. Didn’t even know there was such a thing.</p>

**Comment by vitara-srinivasan** [comment]
<p>why is it classified a minor incident. Been without push ability for &gt; 1 hour</p>

**Comment by Olzie-12** [comment]
<p>I just kept trying every 10 minutes and it looks like everything is working on my side. Pushing and their API. i guess its either the status hasn’t updated or might just be a one off working. Not sure. But webhooks and god damn github actions went down yesterday too.</p>

**Comment by byrneh** [comment]
<aside class="quote no-group" data-username="smontanaro" data-post="5" data-topic="238980" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 smontanaro:</div>
<blockquote>
<p>Oh, cool. Didn’t even know there was such a thing.</p>
</blockquote>
</aside>
<p>Whilst it doesn’t solve your problem, If you do see an issue in the area causing you problem being able to check the status may save you wasting your time investigating.</p>

**Comment by smontanaro** [comment]
<p>Took me a while to get back in front of a computer. <a href="http://githubstatus.com" rel="noopener nofollow ugc">githubstatus.com</a> says only Actions are currently having problems, and my push just worked, so the earlier problems were likely the cause of my errors. I’ll try to remember to use the status website in the future if I encounter problems again.</p>

**Comment by broosted** [comment]
hurry up github support fix it

**Reply by broosted** [reply]
ok its working we are back online
